schema/release-schema.json

```
@@ -617,6 +617,15 @@
           ],
           "format": "date-time"
         },
+        "datePublished": {
+          "title": "Date published",
+          "description": "The date that the award was published.",
```

For consistency with `tender.datePublished`:

```suggestion
          "description": "The date on which the award was published.",
```